htp/htp.h

```diff
@@ -536,6 +536,9 @@ struct htp_tx_t {
 
     /** Transaction index on the connection. */
     size_t index;
+
+    /** Total repetitions for headers. */
+    uint32_t header_repetitions;
```

Comment:
I think I would prefer splitting this into 2: request_header_repetitions and response_header_repetitions. Both could be uint16_t. The benefit I see is that no reset would be needed.

Reply:
To clarify, I'd like to avoid the reset as the flow of the htp state machine is quite obscure even to me as a dev. So keeping things clean would make things easier to follow/debug.

Reply:
You are right :-)